meta-resin-common/recipes-bsp/u-boot/patches/resin-specific-env-integration-kconfig.patch

```diff
@@ -0,0 +1 @@
+NOT IMPLEMENTED YET
```

Comment:
I would add a marker like XXX for grepping in the years to come, if it ever gets forgotten.